proposals/wac-ucr/index.bs

```diff
@@ -911,6 +911,21 @@ write access. Alice grants *PerformChart* read only access to all the projects
 that she can access.
 
 
+### Application determining access privileges {#uc-client-determine-access-privileges}
+
+Guinan uses an application to author and publish documents. The application
+adapts its user interface to distinguish between allowed
+(actionable) and disallowed features based on access information that the
+[=resource server=] reveals, for example, permissions that are granted to the
+application, to the public (everyone), or to a group.
+
+For example, for user level permission, if the application is not granted
+[=write access=] on the resource that Guinan is currently editing, the user
+interface can disable the "Save" button in the menu. Guinan also wants to know
```

Comment:
I would prefer to end this use case here and move "Guinan also wants to know ..." to dedicated use case.